tests/Properties/ByteString.hs

```diff
@@ -49,7 +49,7 @@ import Text.Read
 import Control.Arrow
 import Data.Char
 import Data.Foldable
-import Data.List as L
+import qualified Data.List as L
```

Comment:
```suggestion
import qualified Data.List as List
```

IMHO short abbreviations often hurt readability. In this case the `L` might also stand for "lazy" (bytestrings). But it's not very important to me.

Reply:
Good point, agreed. Done.